### PR TITLE
feat: ability to select network

### PR DESCRIPTION
**Pull request by jonezque**
https://www.notion.so/eb96722b397d42c9908002ca06af88e3?v=35dcf02f76284d8e8efe0b67f7a04234&p=816f1874d8904de6a0da1eb1f4ce3e80

I think Figma design is outdated, network window is in graveyard,
added ability to change current network, tested with arbitrum testnetwork

### CODE REVIEW

**Comment by r0x4xn** [comment]
App <strong>core-dapp</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://core-dapp-cei97s2kb-prepo.vercel.app'>https://core-dapp-cei97s2kb-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_JWVJ3WoV71UcOmB52qrcezjyBL3V"></hidden>

File: apps/frontend/core-dapp/src/components/NetworkDropdown.tsx [hunk 1]
```diff
@@ -72,7 +73,7 @@ const iconNetworkMap: Record<NetworkType, IconName> = {
 }
 
 const comingSoonNetworks: NetworkRef[] = [
-  { iconName: 'arbitrum', name: 'Arbitrum', supported: false, type: 'arbitrum' },
+  { iconName: 'arbitrum', name: 'Arbitrum', supported: false, type: 'arbitrum', chainId: -1 },
```
Comment (r0x4xn) [inline apps/frontend/core-dapp/src/components/NetworkDropdown.tsx:76]: What is `-1` representing here? Can we add it to a constant to be more explicit?

File: apps/frontend/core-dapp/src/components/NetworkDropdown.tsx [hunk 2]
```diff
@@ -100,17 +101,26 @@ const Item: React.FC<{ network: NetworkRef; selectedName: string }> = ({
 
 const NetworkDropdown: React.FC = () => {
   const {
-    web3Store: { network: selectedNetwork },
+    web3Store,
     config: { supportedNetworks },
   } = useRootStore()
+  const { network: selectedNetwork } = web3Store
+
+  const selectNetowrk = (id: ChainId): void => {
```
Comment (r0x4xn) [inline apps/frontend/core-dapp/src/components/NetworkDropdown.tsx:109]: typo `selectNetwork`

**Review by r0x4xn — CHANGES_REQUESTED** [review]

File: apps/frontend/core-dapp/src/components/NetworkDropdown.tsx (hunk 1 above)
Comment (jonezque) [reply apps/frontend/core-dapp/src/components/NetworkDropdown.tsx:76]: we don't yet support Arbitrum,
decided to provide non-existing ID(UI should be blocked for coming soon networks but still)

File: apps/frontend/core-dapp/src/components/NetworkDropdown.tsx (hunk 2 above)
Comment (jonezque) [reply apps/frontend/core-dapp/src/components/NetworkDropdown.tsx:109]: thanks

**Review by r0x4xn — APPROVED** [review]

**Pull request merged**